packages/salesforcedx-utils-vscode/src/telemetry/telemetryReporter.ts

```diff
@@ -177,9 +184,12 @@ export class TelemetryReporter extends Disposable {
         measurements
       });
 
+      const orgId = WorkspaceContextUtil.getInstance().orgId;
+      const properties = { orgId };
       if (this.logStream) {
         this.logStream.write(
           `telemetry/${exceptionName} ${JSON.stringify({
+            properties,
```

Comment:
I'm just following the pattern above of sending a `properties` object to the `logStream.write` method.  Hoping that this just works, but not sure if we need to adjust anything on the Azure side to accept or map the new property?

Reply:
I think it should work :+1: